pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/readers/forward/BaseVarByteChunkForwardIndexReaderV4.java

```diff
@@ -43,9 +43,9 @@
  * (BIG_DECIMAL, STRING, BYTES).
  * <p>For data layout, please refer to the documentation for {@link VarByteChunkForwardIndexWriterV4}
  */
-public class VarByteChunkSVForwardIndexReaderV4
-    implements ForwardIndexReader<VarByteChunkSVForwardIndexReaderV4.ReaderContext> {
-  private static final Logger LOGGER = LoggerFactory.getLogger(VarByteChunkSVForwardIndexReaderV4.class);
+public class BaseVarByteChunkForwardIndexReaderV4
```

Comment:
Rename it to `VarByteChunkForwardIndexReaderV4`. Also update the javadoc for this class

Reply:
Ack